code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -13,6 +13,7 @@
  */
 var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	new/datum/stack_recipe("stool", /obj/structure/chair/stool, one_per_turf = 1, on_floor = 1), \
+	new/datum/stack_recipe("bar stool", /obj/structure/chair/stool/bar, one_per_turf = 1, on_floor = 1), \
```

Comment:
alright i was worried that the addition of the bar stool would mess up the list/make it too long, just wanted to be sure it wouldnt happen